install/kubernetes/Makefile

```diff
@@ -116,12 +119,13 @@ update-versions:
 		# certgen.image.tag												\
 		sed -i '/repository.*certgen.*/{!b;n;s/tag.*/tag: '$(CERTGEN_VERSION)'/}' $(CILIUM_VALUES);			\
 		sed -i 's/'$(CILIUM_PULLPOLICY_REGEX)'/\1 '$$pull_policy'/' $(CILIUM_VALUES);  					\
+		sed -i 's/useDigest:.*/useDigest: '$$use_digest'/' $(CILIUM_VALUES);  							\
 		# image digests;													\
 		sed -i '/# cilium-digest.*/{!b;n;s/digest.*/digest: "'$(CILIUM_DIGEST)'"/}' $(CILIUM_VALUES); 				\
 		sed -i '/# hubble-relay-digest.*/{!b;n;s/digest.*/digest: "'$(HUBBLE_RELAY_DIGEST)'"/}' $(CILIUM_VALUES);			\
-		sed -i '/# operator-aws-digest.*/{!b;n;s/awsDigest.*/awsDigest: "'$(OPERATOR_AWS_DIGEST)'"/}' $(CILIUM_VALUES);                 \
-		sed -i '/# operator-azure-digest.*/{!b;n;s/azureDigest.*/azureDigest: "'$(OPERATOR_AZURE_DIGEST)'"/}' $(CILIUM_VALUES);         \
-		sed -i '/# operator-generic-digest.*/{!b;n;s/genericDigest.*/genericDigest: "'$(OPERATOR_GENERIC_DIGEST)'"/}' $(CILIUM_VALUES); \
+		sed -i '/# operator-aws-digest.*/{!b;n;s/aws-digest.*/aws-digest: "'$(OPERATOR_AWS_DIGEST)'"/}' $(CILIUM_VALUES); 		\
+		sed -i '/# operator-azure-digest.*/{!b;n;s/azure-digest.*/azure-digest: "'$(OPERATOR_AZURE_DIGEST)'"/}' $(CILIUM_VALUES); 	\
+		sed -i '/# operator-generic-digest.*/{!b;n;s/generic-digest.*/generic-digest: "'$(OPERATOR_GENERIC_DIGEST)'"/}' $(CILIUM_VALUES); \
```

Comment:
:thinking: this actually looks wrong to me, I think we may have broken this on master.

Reply:
OK yeah we already fixed it on master (eg commit 8d5c4cb24edc5dec090dd367a7a2d7d28de53dcb). I just need to fix this version to revert this hunk.